src/IconButton/index.js

```diff
@@ -54,6 +54,8 @@ IconButton.propTypes = {
   disabled: PropTypes.bool,
   /** style of icon button to render */
   kind: PropTypes.oneOf(["action", "plain"]),
+  /** type attribute for underlying HTML button element */
+  type: PropTypes.string,
```

Comment:
I could be pressured into making this `PropTypes.oneOf(["button", "submit", "reset"])`. My gut instinct was that, since the valid values are controlled by the HTML gods, not us, I'd rather not be too specific.